# Image Message
string name
# Steps
int32 x_steps           
int32 y_steps           
int32 z_steps        

float32 x_spacing           
float32 y_spacing           
float32 z_spacing           

#sensor_msgs/Image rosimg
# Image data
uint8[] data		  
